openlibrary/templates/support.html

```diff
@@ -52,8 +52,20 @@ <h1>$_("How can we help?")</h1>
 
  $if recaptcha:
    <div class="formElement">
-     <div class="g-recaptcha" data-sitekey="$:recaptcha.public_key"></div>
-   </div>
+      <div class="label smaller lighter">If you have security settings or privacy blockers installed, please disable them to see the reCAPTCHA.</div>
```

Comment:
> privacy blockers

Is this a term?  I doubt that most users would want to install something that blocks their privacy.  I usually hear of popup blockers but perhaps I am behind the times.

Reply:
I am not really sure about this one myself as well, I just copied this https://github.com/internetarchive/openlibrary/blob/8cfe863757f1be2e4353a46e8b9918b113420241/openlibrary/templates/account/create.html#L58
I also think we need to change this from all of the places where it is present 
I searched the file via GitHub search and got two references for this 
Here is the second one 
https://github.com/internetarchive/openlibrary/blob/8cfe863757f1be2e4353a46e8b9918b113420241/openlibrary/i18n/de/messages.po#L405
I think this message will be good @cclauss 
"If you have ad blockers enabled, please disable them to see the reCAPTCHA and refresh the page"
What are your views on this

Reply:
Are we not concerned that patrons might consider that this is asking too much? On a locked-down desktop (such as are common in library kiosks) the patron may not have the luxury of changing security settings. They need another option.

Reply:
@LeadSongDog this might be too much but the librarians were facing issues as stated by Lisa in this comment https://github.com/internetarchive/openlibrary/issues/3826#issue-707526177, I am not sure how else we can resolve this issue